test/reporters/base.spec.js

```diff
@@ -160,6 +168,46 @@ describe('Base reporter', function () {
     });
   });
 
+  describe('unified diff reporter', function () {
+    it('should separate diff hunks by two dashes', function () {
+      var err = new Error('test');
+      var errOut;
+
+      err.actual = createElements({ from: 2, to: 11 });
+      err.expected = createElements({ from: 1, to: 10 });
+      err.showDiff = true;
+      var test = makeTest(err);
+
+      Base.inlineDiffs = false;
+      Base.list([test]);
+
+      errOut = stdout.join('\n');
+
+      var regexesToMatch = [
+        /\[/,
+        /\+ {2}"element 1"/,
+        /"element 2"/,
+        /"element 3"/,
+        /"element 4"/,
+        /"element 5"/,
+        /--/,
+        /"element 7"/,
+        /"element 8"/,
+        /"element 9"/,
+        /"element 10"/,
+        /- {2}"element 11"/,
+        /]/,
+        /test/,
+        /expected/,
+        /actual/
+      ];
+
+      regexesToMatch.forEach(function (aRegex) {
+        errOut.should.match(aRegex);
+      });
```

Comment:
Ouch! My bad; I hadn't noticed those tests before.

In that case I'd be fine merging this as-is for consistency and then doing a separate thorough sweep to correct all instances of that pattern.

Thanks for identifying that!